### DISCUSSION TITLE
Stars rating support

### DISCUSSION THREAD

**Discussion by josefhelie** [ciderapp/Cider#1067] [Feature Request]
Thanks for the great work. I've been testing the software, very neat!!

I fear it's not possible in the web version but I had to ask... is it possible to implement the stars rating? With the automatic playlists, it's the feature that make me stick to Apple Music despite the poor user experience...

**Comment by DCM4711** [comment]
It seems star-ratings are not a thing anymore (at least in Apples Music App). I hope it gets implemented here 😉

**Reply by Tolkana** [reply]
Its still a thing in Apple Music App also, you just need to go to settings. But I also wish it would be implemented in Cider. :)
